frontend/src/bundles/common/components/upload-video/components/video-player/libs/types/player-options.ts

```diff
@@ -0,0 +1,9 @@
+type PlayerOptions = {
+    isPlaying: boolean;
+    wasPlaying: boolean;
+    isMuted: boolean;
+    volume: number;
+    played: number;
```

Comment:
Maybe, it would be better to rename it to timePlayed or secondsPlayed(not sure if it is in seconds)? Because it is not obvious